backend/config.js

```diff
@@ -117,10 +117,13 @@ module.exports = {
       url: process.env.OMIC_URL || 'http://localhost:3001',
     },
     complexity: {
-      url: process.env.COMPLEXITY_OF_NEED_URI,
+      url: process.env.COMPLEXITY_OF_NEED_URI || '',
```

Comment:
To fix `npm run start:dev` - it wouldn't start up out of the box without a default for this